assets/components/src/plugin-settings/SettingsSection.js

```diff
@@ -52,8 +52,18 @@ const getControlType = setting => {
 };
 
 const SettingsSection = props => {
-	const { error, sectionKey, active, title, description, fields, disabled, onChange, onUpdate } =
-		props;
+	const {
+		error,
+		sectionKey,
+		active,
+		title,
+		description,
+		fields,
+		disabled,
+		onChange,
+		onUpdate,
+		sectionSettingsButtonDisabled,
```

Comment:
We're already in the context of the `SettingsSection` component, for readability this could be called `saveDisabled`.

Reply:
I have added the suggestion. Thanks!